database.py

```diff
@@ -0,0 +1,219 @@
+"""
+This Module is Where all Database actions are done, which is used
+by the user.py module to get user Data.
+
+The program uses sqlite3 for a simple and great Database for storing,
+and changing data in the Person.db local file.
+
+To create a user the new_person method is used, and to check there is a user
+with the name, check_if_user_exists method is used.
+there are several get_* and change_* methods:
+
+get_days, get_primary_color, get_secondary_color, get_days_goal and 
+get_motivation.
+
+as their names say they simply selects and gets the information from the 
+database
+
+change_days, change_primary_color, change_secondary_color, change_days_goal
+and change_motivation
+
+again, as their names say they select and change the information in the database
+"""
+import sqlite3
+class DataBaser(object):
+    """Contains all things related to change data or get data on the DataBase
+       
+    Attributes:
+        cdb: is the Connector to the DataBase,it creates a new file if it
+    doesnt exists as well
+    """
+    def __init__(self):
+        """Connects to the DataBase
+        creates a Table, if it exists uses try and except so error doesnt raise
+        """
+        self.cdb = sqlite3.connect('PersonDB.db')
+        try: 
+            self.cdb.execute('''CREATE TABLE PERSON
+                     (NAME TEXT NOT NULL,
+                     DAYS  INT  NOT NULL,
+                     PRIMARYCOLOR  TEXT  NOT NULL,
+                     SECONDARYCOLOR TEXT NOT NULL,
+                     DAYSGOAL INT NOT NULL,
+                     MOTIVATION TEXT NOT NULL);''')
+            self.cdb.commit()
+        except sqlite3.OperationalError:
+            pass
+
+    def new_person(self, person_name):
+        """Creates a New Person to the Person Table With the Default Values
+           Args:
+               person_name: The Name of the Person that needs to be recorded
+           in the Table
+        """
+        #Default Values:
+        primary_color = "#0000FF"
+        secondary_color = "#008000"
+        motivational_phrase = "Put Motivational phrases Here"
+        self.cdb.execute("INSERT INTO PERSON (NAME, DAYS, PRIMARYCOLOR,\
+                                             SECONDARYCOLOR, DAYSGOAL,\
+                                             MOTIVATION) VALUES ('%s',\
+                                             0, '%s',\
+                                            '%s', 100,\
+                                            '%s')"%(person_name,
+                                            primary_color, secondary_color,
+                                                    motivational_phrase))
+        self.cdb.commit()
+    def check_if_user_exists(self, person_name):
+        """Check if User exists in The DataBase
+        Args:
+            person_name: The Name of the person that will be searched in the DB
```

Comment:
It only returns true because if the user name doesn't exists it creates a new one,  I didn't set a return docstring but it could be added for better understanding